### PR TITLE
Driver post ride

### PR DESCRIPTION
**Pull request by whyderrick**

### CODE REVIEW

File: app/controllers/rides_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class RidesController < ApplicationController
+  def index
+    @rides = Ride.all
+    respond_to do |format|
```
Comment (bootcoder) [inline app/controllers/rides_controller.rb]: Pretty sure the index works without the respond to block. Try it....

File: app/controllers/rides_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      respond_to do |format|
+        format.html { render :new }
+        format.json { render json: @ride.errors, status: :unprocessable_entity}
+      end
+    end
+
+  end
+
+  def show
+    @ride = Ride.find(params[:id])
```
Comment (bootcoder) [inline app/controllers/rides_controller.rb]: DRY this out with a before_action ala
```ruby 
  before_action :set_user, only: [:show, :edit, :update, :destroy]
```
followed by
```ruby 
  def set_user
    @user = User.find(params[:id])
  end
```

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
```
Comment (bootcoder) [inline app/controllers/users_controller.rb]: You're not doing anything with this controller nor is it your responsibility.......
rm -rf

File: app/views/rides/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+%h1 Rides index
+
+- @rides.each do |item|
```
Comment (bootcoder) [inline app/views/rides/index.html.haml]: IMHO ul-li is the better tag here

**Review by bootcoder — COMMENTED** [review]
Couple things, mostly awesome. ☯️

**Comment by whyderrick** [comment]
Merging in after addressing PR comments.

**Pull request merged**